src/main/java/mate/controller/GetAllCarsController.java

```diff
@@ -0,0 +1,26 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/all")
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/cars")
```
it is better practice to write just `/cars` - same for drivers and manufacturers

Reply:
edited ✔️